app/models/booking.rb

```diff
@@ -44,6 +44,11 @@ class Booking < ApplicationRecord
   scope :by_status, ->(status){where status: status if status.present?}
   scope :by_user_id, ->(id){where user_id: id}
   scope :order_by_status, ->{order :status}
+  scope :get_data_chart, (lambda do
```

Comment:
sao e k dùng luôn scope by_status, cần gì viết cái ms
truyền vào 1 mảng nó sẽ tự động where in

Reply:
vâng ạ